recruiter/tests.py

```diff
@@ -60,9 +60,6 @@ def test_get_all_companies_returns_all_companies_as_a_query_set():
     companies_set = Company.get_all_companies()
     assert isinstance(companies_set, QuerySet)
     assert all(isinstance(company, Company) for company in companies_set)
```

Comment:
If we drop the assertions below, how do we know that we're getting the valid company data and not some garbage?

Reply:
It just seems to me a bit "dirty" code to check each name one by one as it was

Reply:
We check if we get back an company instance's

Reply:
What guarantee do you have that the instance fields contain proper data? Maybe a string decoder got messed up somewhere?
Also what about the order of responses? You were checking that as well with the comparison before.

Reply:
ok, done.